samza-sql/src/main/java/org/apache/samza/sql/interfaces/SqlIOConfig.java

```diff
@@ -81,11 +81,13 @@ public SqlIOConfig(String systemName, String streamName, List<String> sourcePart
     this.streamId = String.format("%s-%s", systemName, streamName);
 
     samzaRelConverterName = streamConfigs.get(CFG_SAMZA_REL_CONVERTER);
-    Validate.notEmpty(samzaRelConverterName, String.format("System %s is not supported. Please check if the system name is provided correctly.", systemName));
```

Comment:
Can you revert the change for validation here and below ?

Reply:
Done.